website/code_issues.qmd

```diff
@@ -383,4 +383,44 @@ CRAN doesn't allow negative `warn` options. This setting will turn off all warni
 CRAN recommends using `suppressWarnings()`, which disables warnings only for the specific expression it's applied to, rather than globally.
 
 
+-----------------------------------------
+
+# Using more than 2 Cores
+
+## Problem
+
+You are using more than 2 CPU cores in your examples, vignettes, tests or during installation. 
+
+## Solution
+
+Make sure that you set the maximum number of cores used to 2 in examples, vignettes or tests. 
+During installation, many modern build systems use the `-j` flag. 
+<!-- This was recommended by 'aitap' in a discussion on GitHub -->
+
+### Details
+
+::: {.callout-note title="CRAN Review Communication" appearance="simple" icon=false collapse=true}
+
+Please ensure that you do not use more than 2 cores in your examples, vignettes, etc.
+
+:::
+
+
+CRAN checks run on big, multi-core computers, running many `R CMD check` processes in parallel. Every process is allowed to use a maximum of two CPU cores to exercise parallel code in examples and unit tests. When using more than 2 cores, your package will get a NOTE from CRAN checks, saying that it took more CPU time than elapsed time.
```

Comment:
For consistency can you update the 'two CPU cores' to '2 CPU cores' to match the rest of the recipe?